candi/openapi/cluster_configuration.yaml

```diff
@@ -126,7 +126,8 @@ apiVersions:
       proxy:
         type: object
         description: |
-          Global proxy setup (especially for working in air-gapped environments).
+          Global proxy setup (especially for working in air-gapped environments). The parameters described in this section will be translated into the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` for all cluster nodes and Deckhouse components.
+          This will lead to HTTP/S requests (curl, git, registry, etc.) to all resources not specified in the noProxy parameter being executed through a proxy. The `podSubnetCIDR`, `serviceSubnetCIDR` subnets and the `clusterDomain` domain are added to noProxy automatically.
```

Comment:
```suggestion
          Global proxy setup (especially for working in air-gapped environments).
          
          The parameters described in this section will be translated into the environment variables `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` for all cluster nodes and Deckhouse components. This will lead to HTTP(S) requests (curl, git, registry, etc.) to all resources not specified in the `noProxy` parameter being executed through a proxy. The `podSubnetCIDR`, `serviceSubnetCIDR` subnets, and the `clusterDomain` domain are added to `noProxy` automatically.
```